go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ONSdigital/dp-cantabular-csv-exporter
 
-go 1.18
+go 1.19
```

Comment:
I wonder if we need to update the `Dockerfile.local`?

Reply:
Thanks rafa
Your observation is now applied